include/grpc/impl/codegen/port_platform.h

```diff
@@ -105,31 +105,6 @@
 #define GPR_WINDOWS_ATOMIC 1
 #define GPR_MSVC_TLS 1
 #endif
-#elif defined(GPR_MANYLINUX1)
```

Comment:
Do we know which features this will enable for manylinux2010 artifacts? I'm particularly interested in whether this will resolve https://github.com/grpc/grpc/issues/18210.

Reply:
Technically all Linux features introduced between Kernel 2.6.32 and 3.10 would be the answer. But I don't know which are directly related to gRPC.